bouncer/src/repo/core/model/bson/repo_node_mesh.cpp

```diff
@@ -173,7 +173,19 @@ RepoNode MeshNode::cloneAndApplyTransformation(
 	}
 }
 
-MeshNode MeshNode::cloneAndUpdateMeshMapping(
+SupermeshNode SupermeshNode::cloneAndUpdateIds(
```

Comment:
I think this should live in RepoNode

as these fields are common for any repo nodes (and I feel like we already have something similar in that class... not 100% sure though)

Reply:
Hi @carmenfan, the problem with putting this in `RepoNode` is that that will return the base class, whereas the caller expects a `SupermeshNode`. However I've thought again and a neater way is just to not have it and make `buildMesh` take the Ids, which is closer to the goal of treating meshes as immutable once built anyway!